charts/llm-inference-app/values.yaml

```diff
@@ -104,15 +104,8 @@ amdImage:
   pullPolicy: IfNotPresent
 
 envAmd:
-  HIP_VISIBLE_DEVICES: "0"
```

Comment:
this essentially breaks installations on multi-GPU nodes where GPU `0` is already occupied.